### PR TITLE
Change repository references to https://github.com/kubernetes/kubernetes

### PR DESCRIPTION
**Pull request by mamikonyana**
I couldn't reopen #18595 (github wouldn't let me) so I'm creating a new pull request
@mikedanese please have a look

### CODE REVIEW

File: Godeps/_workspace/src/github.com/spf13/cobra/bash_completions.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ cmds := &cobra.Command{
 	Short: "kubectl controls the Kubernetes cluster manager",
 	Long: `kubectl controls the Kubernetes cluster manager.
 
-Find more information at https://github.com/GoogleCloudPlatform/kubernetes.`,
+Find more information at https://github.com/kubernetes/kubernetes.`,
```
Comment (mikedanese) [inline Godeps/_workspace/src/github.com/spf13/cobra/bash_completions.md]: This is going to break a precommit check (travis will be red). We should leave Godeps/ as is.

**Comment by mikedanese** [comment]
LGTM other than godeps. @k8s-bot ok to test

**Comment by mamikonyana** [comment]
@mikedanese I updated the commit. Out of curiosity why did changing the reference in Godeps break travis?

**Comment by brendandburns** [comment]
@k8s-bot test this please

**Comment by mikedanese** [comment]
@mamikonyana It fails because we verify the godeps directory is reproducibly creatable from the git refs we have our dependencies pinned to. If we make changes to Godeps/ that are not merged into the upstream project, the precommit breaks.

**Comment by mikedanese** [comment]
@k8s-bot e2e test this please

**Comment by mamikonyana** [comment]
@mikedanese thanks for the insight.

**Pull request merged**